### DISCUSSION TITLE
software currently not available for software server

### DISCUSSION THREAD

**Discussion by jekijekicom** [Homebrew/discussions#81] [Everyday usage]
i tried to download homebrew, while it goes to finding software there's notif says like the picture below. what should i do? 
![Screen Shot 2020-10-21 at 11 57 07](https://user-images.githubusercontent.com/57800593/96675523-3c709c80-1395-11eb-95bd-debe295e3cbc.png)

**Comment by SMillerDev** [comment]
Since you're running beta software you should install the latest beta of the commandline tools from developer.apple.com
